### PR TITLE
feat: added text utilities and lists

### PR DESCRIPTION
**Pull request by HeyNitin**
image

### CODE REVIEW

**Comment by ajit-sharmaGH** [comment]
Hey i went through your code, looks fine to merge 😎

File: Components/Text Utilities/text utils.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  font-size: 3rem;
+}
+
+.small-text {
+  font-size: 1.5rem;
+}
+
+.gray-area {
+  color: var(--grey-color);
+}
```
Comment (rohanmathur91) [inline Components/Text Utilities/text utils.css:13]:
``` suggestion
Is this for text? if yes then you can name it gray or gray-text.
````

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

File: Components/Text Utilities/text utils.css (hunk 1 above)
Comment (HeyNitin) [reply Components/Text Utilities/text utils.css:13]: Yes, will surely do. 
Thanks for reviewing Rohan

File: Components/Text Utilities/text utils.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@import "../../Colors/colors.css";
+
+.heading {
```
Comment (surbhi84) [inline Components/Text Utilities/text utils.css:3]: Headings can be of different sizes try to implement more classes for those sizes as well moreover only defining fonts size is not enough try incorporating more properties.

File: Components/Text Utilities/text utils.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  font-size: 3rem;
+}
+
+.small-text {
```
Comment (surbhi84) [inline Components/Text Utilities/text utils.css:7]: 1.5rem seems a lot for small text you can decrease it to a more appropriate value.

**Comment by surbhi84** [comment]
I believe you should consider making these changes.

File: Components/Text Utilities/text utils.css (hunk 3 above)
Comment (HeyNitin) [reply Components/Text Utilities/text utils.css:7]: Will update in next PR. Thanks for the suggestions captain 🤝

File: Components/Text Utilities/text utils.css (hunk 2 above)
Comment (HeyNitin) [reply Components/Text Utilities/text utils.css:3]: On it...

**Pull request merged**